client/public/css/styles.css

```diff
@@ -0,0 +1,8 @@
+body {
+  background: #f1f1f1;
+}
+.login {
+  width: 320px;
+  padding: 8% 0 0;
+  margin: auto;
+}
```

Comment:
The style css is called from index.css in the src folder. Please add the styles on that file